mm2src/coins/utxo/utxo_common.rs

```diff
@@ -50,6 +50,9 @@ lazy_static! {
 }
 
 pub const HISTORY_TOO_LARGE_ERR_CODE: i64 = -1;
+/// `get_sender_trade_fee` and `get_receiver_trade_fee` should take into account that dynamic fee may increase during the swap.
+/// So we should increase the dynamic fee by 1%.
+pub const TRADE_PREIMAGE_DYNAMIC_FEE_PERCENT: f64 = 1.;
```

Comment:
Yesterday my comment hasn't been posted somehow :|
I've run a test for a while. And I found out the dynamic fee varies within 0.1 percent per block. I think if we set this value to 10%, it will be too much

Reply:
During the discussion in person, we agreed that 1% is sufficient. If we increase it too much, we can get some dust left on the address in the max volume trade case.